releases/2.12.18.md

```diff
@@ -0,0 +1,46 @@
+# Scala 2.12.18
+
+This release improves **compatibility**:
+
+* Support JDK 20 with ASM 9.5 upgrade, by @Philippus ([#10185](https://github.com/scala/scala/pull/10185), [#10362](https://github.com/scala/scala/pull/10362))
+
+It also aids **migration** to Scala 2.13 and Scala 3:
+
+* Under `-Xsource:2.13`, warn when inherited takes precedence over outer definitions ([#10297](https://github.com/scala/scala/pull/10297), [#10321](https://github.com/scala/scala/pull/10321))
+* Under `-Xsource:2.13`, flag eta-expanding methods without parameter lists ([#10283](https://github.com/scala/scala/pull/10283))
+* Under `-Xsource:2.13`, don't GLB binders of type patterns, use the type directly ([#10298](https://github.com/scala/scala/pull/10298))
+
+and improves **linting**:
+
+* Lint for integral divisions that are widened to a float ([#10313](https://github.com/scala/scala/pull/10313))
+
+and enables improved **tooling**:
+
+* Preserve tree attachment in the typer's `Block`/`Apply` inversion transform ([#10176](https://github.com/scala/scala/pull/10176))
+
+For complete 2.12.18 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.12.18) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.12.18).
```

Comment:
"all fixed bugs" or "all closed issues" or "tickets". I can't tell if that is pedantic or clarifying.

Reply:
I've always hesitated around this wording because some PRs fix bugs that were never reported to scala/bug. The “closed” is a nod to that. But maybe that little nod isn't actually meaningful but anybody but myself 🤷